modules/acquia_cms_common/modules/acquia_cms_support/src/Service/AcquiaCmsConfigSyncService.php

```diff
@@ -70,33 +70,27 @@ public function __construct(
    * @param \Drupal\Core\Config\StorageInterface $syncStorage
    *   The storage to use as sync storage for compairing changes.
    *
-   * @return float
-   *   parity between configuration.
+   * @return int
+   *   Parity value between configuration.
    */
-  public function getParity($configFile, StorageInterface $syncStorage) {
+  public function getParity($configFile, StorageInterface $syncStorage): int {
     // Database configuration.
-    $active_configuration = explode("\n", Yaml::encode($this->targetStorage->read($configFile)));
+    $activeConfiguration = explode("\n", Yaml::encode($this->targetStorage->read($configFile)));
     // Configuration in files.
-    $original_configuration = explode("\n", Yaml::encode($syncStorage->read($configFile)));
-    $active_configuration = $this->removeNonRequiredKeys($active_configuration);
-    $diff = new Diff($original_configuration, $active_configuration);
-    $totalLines = count($active_configuration);
+    $originalConfiguration = explode("\n", Yaml::encode($syncStorage->read($configFile)));
+    $activeConfiguration = $this->removeNonRequiredKeys($activeConfiguration);
+    $diff = new Diff($originalConfiguration, $activeConfiguration);
```

Comment:
I think this is deprecated in 10.1 - https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Diff%21Diff.php/function/Diff%3A%3AisEmpty/10
We need to bring the Diff class locally and may be remove in future if we have any alternative.

cc @vishalkhode1